scripts/node_power_control.py

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""script to control power of node elements (snaps, fem, pam)
+
+"""
+from __future__ import absolute_import, division, print_function
+
+import sys
+from astropy.time import Time
+
+from hera_mc import mc, cm_utils
+
+if __name__ == '__main__':
+    parser = mc.get_mc_argument_parser()
+    parser.description = """Power on or off node elements (snaps, fem, pam)"""
+    parser.add_argument('nodes', help="list of nodes to command (integers) (comma-separated,no spaces) or 'active'")
+    parser.add_argument('parts', help="list of part to command (comma-separated,no spaces) or 'all'.")
```

Comment:
Also, I think it might end up being helpful if the help text could include a list of valid part names — seems like the kind of thing that a user might have trouble remembering.

Reply:
good idea, I added that list.